docs/finalproject/week-3/lesson.md

```diff
@@ -15,3 +21,21 @@ It's important to evaluate our contributions to the group. Here's what we expect
 - A full stack developer must deliver features that touch each part of the stack. This means at minimum you must build a component of UI that interacts with the database and you must configure and manage that entire interaction pathway. At best this means multiple features across different parts of the stack.
 
 For graduation, we evaluate each role against the Skills for the Information Age skills descriptions, which you should check for more details.
+
+
+### Cooling periods during final projects 
+
+Cooling periods are for teams that are struggling to collaborate, communicate or work together in an agile way. This is an indication that the team may not be on track to graduate due to concerns about their professional skills. 
```

Comment:
It may be worth explicitly mentioning that this isn't a failure, that often times in real life teams have communication issues, and that working out how to resolve them is an important step?

(But can definitely be a follow-up change!)

Reply:
Oh actually, I just realised this is in week 3... Is this something we want to introduce earlier? It feels like "if you're struggling to collaborate, we should work out how to fix it" should be something we introduce early?

Reply:
Agree, if there is a possibility of not graduating I think we should be communicating possibility that up front. (Apologies if that's already the case! I haven't had time to keep track of the changes)

I think it's fine to have some more detailed wording/actionable exercises about it here.

Reply:
@40thieves , if people cannot graduate, they won't even start Final Project. Once they start the FP we track how they are doing (commit-wise), and we also give them support on their conflict bit. 

@illicitonion , I am happy to move it to week 2 (on week 1 we have the main presentation that already talks about conflicts). I just used week 3 since it was an empty page :D 

Let me know your thoughts and I'll change it.

Reply:
Yeah, let's move this as early as we can - week two sounds great. Maybe we can frame this as either part of, or an alternative to, the Retrospective section currently in week 2? The steps are actually pretty similar to what's already outlined as the Retrospective section, just with a more focused problem and more guidance towards solutions...

Reply:
> if people cannot graduate, they won't even start Final Project

I think I was reacting to the wording here that says they may not graduate if they can't communicate (to be clear, I think this is a good thing!) I just think if we don't mention that there's a possibility of not graduating due to comms problems _during the final project_, then we should discuss that early.

But it sounds like we're reaching the same conclusion - I'm happy to move it to the week 2 as @illicitonion suggests.